src/SloganForm.js

```diff
@@ -1,24 +1,29 @@
-export default function SloganForm(props) {
+import { useState } from 'react';
+
+export default function SloganForm({ setSlogan, slogan }) {
   // React forms are a pain! 
   // Track the sloganInput form state with a useState hook
+  const [sloganInForm, setSloganInForm] = useState('');
     
   function handleSubmit(e) {
     e.preventDefault();
 
     // This component takes in a prop called setSlogans, which is a function that takes in an array of slogans. 
+    setSlogan([...slogan, sloganInForm]);
     // Also, this component also takes in a prop called slogans, which is an arry of strings. //// On submit call the setSlogans state with a new array that is a copy of the old array with the new slogan immutably added to the end 
 
     // set the sloganInput form state to an empty string to reset the form
+    setSlogan('');
```

Comment:
Here you would want to `setSloganInForm` to an empty string.  As it stands you are undoing your previous state change, which I think is why slogans are not updating/rendering on the page.